.github/workflows/reusable-build.yml

```diff
@@ -246,15 +256,18 @@ jobs:
               COMMIT_TAGS+=("${SHA_SHORT}")
           fi
 
+          TODAY="$(date +%A)"
           if [[ ${{ matrix.fedora_version }} == "stable" ]]; then
```

Comment:
This is much more readable now.

That said I think we have room for improvements in the future.